### PR TITLE
Add 2019-12-16 NumPy Community Survey Meeting minutes

### PR DESCRIPTION
**Pull request by InessaPawson**
Adding 2019-12-16 NumPy Community Survey Meeting minutes.

### CODE REVIEW

File: status_meetings/survey_2019-12-16.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+2. Presentation at UM Computer Science and Engineering Department (topic, presentation, slide deck).
+ - Contacted 2 EECS POC's, waiting to hear back
+ - Proposed topics:
+   * A little history
+   * Scientific python ecosystem (and NumPy's place in it)
+   * Recent developments
+   * Future (near & far-term)
+ - Slides
+   * **Very rough** outline [here](https://gist.github.com/rossbar/24161df0370e95f11090011a1bcfa4f6)
```
Comment (rgommers) [inline status_meetings/survey_2019-12-16.md]: this link doesn't work, and I don't see any such document on @rossbar's gists. fix, or remove?

**Comment by rgommers** [comment]
Very nice, thanks @InessaPawson and @rossbar, looking forward to seeing where this goes!

File: status_meetings/survey_2019-12-16.md (hunk 1 above)
Comment (InessaPawson) [reply status_meetings/survey_2019-12-16.md]: I can see that the link in the meeting minutes on HackMD was updated after I submitted this PR. I've updated the link in this PR.

**Comment by rgommers** [comment]
Merged, thanks Inessa

**Pull request merged**